### ISSUE TITLE
POS allowed Mood=Irr

### ISSUE THREAD

**Issue by NathanD38** [IAHLT/UD_Hebrew#61]
@amir-zeldes

In the following contra-factual condition,

אילו משה היה מגיע לוועדה, הוא היה עומד בתנאים לקבלת דמי אבטלה.

the protasis includes the sequence היה מגיע, in which היה assigns Tense=Past,
and מגיע receives Mood=Irr without Tense.

The apodosis includes the progressive היה עומד, in which, again, היה assigns Tense=Past,
and עומד receives Aspect=Prog without Tense.

I assume that the analysis is consistent if we have a past tense VERB with the same meaning of היה מגיע:

לו/(אילו) משה הגיע לוועדה, הוא היה עומד בתנאים לקבלת דמי אבטלה.

Because we have a contra-factual condition, the past tense verb should receive Mood=Irr in addition to its Tense=Past.

Now, we have found in our data the following sentence:

השכר בעבודה המוצעת שווה לפחות לדמי האבטלה ש**היו מגיעים לדורש העבודה אילו היה זכאי להם**

The apodosis precedes the protasis here, and we know that היו מגיעים would be analyzed the same as the example above.
What should we do in the protasis? We have היה which is assigned AUX+cop (VerbType=Cop) because it precedes an ADJ, זכאי.
Should ADJ be allowed Mood=Irr, since clearly this is a contra-factual condition?

What would be the case of complex modal constructions?

לו משה היה זכאי לקבל דמי אבטלה, הוועדה לא הייתה דוחה את דרישתו.

Actual examples, so I wouldn't be accused of inventing these abominations:

עבור יום ההיעדרות השני ויום ההיעדרות השלישי בכל סדרת טיפולים, העובדת זכאית לתשלום דמי מחלה 
בשווי של 50% מהשכר ש**היתה זכאית לקבל אלמלא נעדרה מהעבודה בכל אחד מהימים הללו.**

תגמול המילואים יהיה לפחות בשיעור דמי האבטלה ש**המובטל היה מקבל לולא יצא למילואים** (על המובטל לדווח ללשכת שירות התעסוקה על שירותו במילואים, כדי שלא יירשמו לו בטעות ימי אבטלה בזמן שירות המילואים).

[which is also a case for לולא and אלמלא being SCONJ, and perhaps with Polarity=Neg, which is currently not allowed
by our validator. But maybe that's a topic for a separate issue.]

Here, as per issue #48, היה is aux+AUX (does not receive VerbType=Cop), and זכאי is a modal aux+AUX (VerbType=Mod),
and לקבל is our predicate VERB in the protasis. Only היה can assign Tense here. 
Since לקבל is an infifnitive, it cannot receive Tense, but can it receive Mood=Irr, 
seeing that it occupies the same slot of a contra-factual verb?

**Comment by amir-zeldes** [comment]
> Should ADJ be allowed Mood=Irr

I would probably fold it into the copula (I think mood on an auxiliary is not so odd). נעדרה/יצא is a finite verb so that one is easy (Irr). I would also support Polarity=Neg on the negative conditional items.

I don't think we've ruled on infinitive mood yet, but TBH I see a case to be made that the auxiliary should carry it either way... But if we have a long standing precedent letting the participle carry the mood of the construction (since it is the lexical verb head), then maybe it's consistent to do the same with infinitives.

**Comment by NathanD38** [comment]
@amir-zeldes 

Thanks! I think to keep it consistent with previous practice, we should leave the infintives tagged VERBs with Mood,
but ask Nick to allow היה to receive Mood=Irr when it is AUX+cop, i,e,, followed by an ADJ.

Is that ok?

**Comment by amir-zeldes** [comment]
Yeah, I think for now this is fine (UD still needs to clarify the distinction between moods applying to a morphological form and a construction, but we are not going to solve that immediately, and as long as it's marked on one of them we can always figure it out later)
